cypress/utils/signup.js

```diff
@@ -4,8 +4,12 @@ import login from "./login";
 export default function signup() {
```

Comment:
Tests are failing

Reply:
We shouldn't need to add the db var since it's already in ui/.env, but also tests were passing before.

Reply:
Or they were never fully working, my bad. But either way.